docs/platform/porting/recovery.md

```diff
@@ -36,492 +36,10 @@ the `/system` directory, and other filesystems (such as `/opt` and
 `/opt/usr`) below the `/system` directory. The script starts a shell on the serial console and launches the
 `system-recovery` program.
 
-4. The `system-recovery` program can work as either an interactive GUI or
-a non-interactive dispatcher, executing 1 (non-interactive) or
-more (interactive) actions. The actions are configured in the
-`/usr/share/system-recovery/system-recovery.cfg` file.
-
-## Configuration File
```

Comment:
Are lines 44 to 524 also removed in the new commit?

Reply:
Yes, that section is describing old configuration which is not used current.